spynnaker/pyNN/models/neuron/synapse_types/synapse_type_alpha.py

```diff
@@ -53,12 +53,24 @@ class SynapseTypeAlpha(AbstractSynapseType):
     def __init__(self, exc_response, exc_exp_response,
                  tau_syn_E, inh_response, inh_exp_response, tau_syn_I):
         r"""
-        :param float exc_response: :math:`response^\mathrm{linear}_e`
-        :param float exc_exp_response: :math:`response^\mathrm{exponential}_e`
-        :param float tau_syn_E: :math:`\tau^{syn}_e`
-        :param float inh_response: :math:`response^\mathrm{linear}_i`
-        :param float inh_exp_response: :math:`response^\mathrm{exponential}_i`
-        :param float tau_syn_I: :math:`\tau^{syn}_i`
+        :param exc_response: :math:`response^\mathrm{linear}_e`
+        :type exc_response: Float, iterable of Floats, RandomDistribution or \
+                            function
+        :param exc_exp_response: :math:`response^\mathrm{exponential}_e`
+        :type exc_exp_response: Float, iterable of Floats, \
+                                RandomDistribution or function
+        :param tau_syn_E: :math:`\tau^{syn}_e`
+        :type tau_syn_E: Float, iterable of Floats, RandomDistribution or \
+                         function
+        :param inh_response: :math:`response^\mathrm{linear}_i`
+        :type inh_response: Float, iterable of Floats, RandomDistribution or \
+                            function
+        :param inh_exp_response: :math:`response^\mathrm{exponential}_i`
+        :type inh_exp_response: Float, iterable of Floats, \
+                                RandomDistribution or function
+        :param tau_syn_I: :math:`\tau^{syn}_i`
+        :type tau_syn_I: Float, iterable of Floats, RandomDistribution or \
+                         function
```

Comment:
Is there a reason why you've added the params/types like this for the components (i.e. synapse_type, input_type, threshold_type), but not for the neuron models?  I think it's just neuron_model_izh.py and neuron_model_leaky_integrate_and_fire.py where you've not done it...

Reply:
ermm........... no idea what ya on about ;-) definitely wasn't that i missed those completely until you pointed them out to me and fixed. definitely not that. fixed now.